fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -1053,6 +1053,23 @@ public void countKeyValue(@Nonnull final FDBStoreTimer.Count key,
                 primaryKey, metaData.isSplitLongRecords(), omitUnsplitRecordSuffix);
     }
 
+    @Nonnull
+    private Range getRangeForRecord(@Nonnull Tuple primaryKey) {
+        return TupleRange.allOf(primaryKey).toRange(recordsSubspace());
```

Comment:
Note that as a consequence of this, this *also* adds a conflict range for all records of the given prefix. I'm not sure if this is something to advertise or not.